doc/whats_new/v1.0.rst

```diff
@@ -45,6 +45,13 @@ Changelog
     where 123456 is the *pull request* number, not the issue number.
 
 
+:mod:`sklearn.feature_extraction`
+..................................
+
+- |Fix| Fixed a bug in :func:`image._to_graph` where singleton
+    connected components were not handled properly, resulting in a wrong
+    vertex indexing.  :pr:`18964` by `Bertrand Thirion`_.
```

Comment:
```suggestion
- |Fix| Fixed a bug in :func:`image._to_graph` where singleton
  connected components were not handled properly, resulting in a wrong
  vertex indexing. :pr:`18964` by `Bertrand Thirion`_.
```

Reply:
Actually, it should refer to `img_to_graph` and `grid_to_graph` instead of the private `_to_graph`.